CorsixTH/Src/th_lua.h

```diff
@@ -227,8 +227,8 @@ template <> struct luaT_classinfo<layers> {
     static inline const char* name() {return "Layers";}
 };
 
-class THPathfinder;
-template <> struct luaT_classinfo<THPathfinder> {
+class pathfinder;
```

Comment:
`path_finder` is perhaps better in line with the other code??

Reply:
Pathfinder is one word in English.